packages/typespec-go/test/azure-http-specs/azure/resource-manager/commonpropsgroup/zz_models.go

```diff
@@ -6,6 +6,12 @@ package commonpropsgroup
 
 import "time"
 
+// ErrorResponse - A response containing error details.
+type ErrorResponse struct {
```

Comment:
This shows up because the type has flag `Error` instead of `Exception` set. Is this expected?

Reply:
i think so. after tcgc remove the `Error` usage, then the model will be removed from the model list. i refined the model filtering logic to only keep the model with input or output usage. that could also help to filter out the exception and lro middle step models.